hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1090,15 +1090,41 @@
     <value>100</value>
     <tag>OZONE, MANAGEMENT, PERFORMANCE</tag>
     <description>
-      The number of RPC handler threads for each SCM service
-      endpoint.
-
-      The default is appropriate for small clusters (tens of nodes).
-
-      Set a value that is appropriate for the cluster size. Generally, HDFS
-      recommends RPC handler count is set to 20 * log2(Cluster Size) with an
-      upper limit of 200. However, SCM will not have the same amount of
-      traffic as Namenode, so a value much smaller than that will work well too.
+      Used to set the default number of handler threads for
+      RPC of the SCM service.
+      If you need to set different handlers for a specific RPC,
+      you can refer to the following mapping relationship:
+          ---- RPC type ---- : ---- Configuration items ----
+      SCMClientProtocolServer: 'ozone.scm.client.handler.count.key'
+      SCMBlockProtocolServer: 'ozone.scm.block.handler.count.key'
+      SCMDatanodeProtocolServer: 'ozone.scm.datanode.handler.count.key'
```

Comment:
How about we revise the paragraph as below?
```
      To specify handlers for individual RPC servers,
      set the following configuration properties instead:
      ---- RPC type ----       : ---- Configuration properties ----
      SCMClientProtocolServer  : 'ozone.scm.client.handler.count.key'
      SCMBlockProtocolServer   : 'ozone.scm.block.handler.count.key'
      SCMDatanodeProtocolServer: 'ozone.scm.datanode.handler.count.key'
```
The entire description will look like
```
      The number of RPC handler threads for each SCM service
      endpoint.

      The default is appropriate for small clusters (tens of nodes).

      Set a value that is appropriate for the cluster size. Generally, HDFS
      recommends RPC handler count is set to 20 * log2(Cluster Size) with an
      upper limit of 200. However, Ozone SCM will not have the same amount of
      traffic as HDFS Namenode, so a value much smaller than that will work well too.

      To specify handlers for individual RPC servers,
      set the following configuration properties instead:
      ---- RPC type ----       : ---- Configuration properties ----
      SCMClientProtocolServer  : 'ozone.scm.client.handler.count.key'
      SCMBlockProtocolServer   : 'ozone.scm.block.handler.count.key'
      SCMDatanodeProtocolServer: 'ozone.scm.datanode.handler.count.key'
```